src/test/fuzz/rbf.cpp

```diff
@@ -23,7 +23,15 @@ namespace {
 const BasicTestingSetup* g_setup;
 } // namespace
 
+// Maximum number of things we will effect during an RBF
```

Comment:
this is hopefully a conservative over-estimate since clusters should only be 101kvB total each